wordpress-mobile/WordPress-iOS
# Fix Aztec placeholder on Gutenberg

This PR updates the Gutenberg reference to test https://github.com/wordpress-mobile/gutenberg-mobile/pull/591

I also took the opportunity to try to improve our gutenberg/rn-aztec Podfile integration, adding a function that modifies both at the same time. So now we just have to modify the hashes, tags, paths, branches, etc... in just one place.

To test:
- `rake dependency`
- Check that both gutenberg and rn-aztec are updated appropriately.

To test the placeholders, please refer to https://github.com/wordpress-mobile/gutenberg-mobile/pull/591
